app/APIStart.py

```diff
@@ -8,148 +8,206 @@
 from flask import Flask
 from flask import request
 from flask import jsonify
-import json
-from DBhandler import DatabaseHandler as db
-from Items import Item as item
-from User import User
+from app.User import User, UserBase
```

Comment:
According to pep8: 

Imports should be grouped in the following order:

> Standard library imports.
> Related third party imports.
> Local application/library specific imports.

You should put a blank line between each group of imports.

Reply:
Ok Got It.